pkgs/development/tools/pass-git-helper/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, isPy3k, fetchFromGitHub, pyxdg }:
+
+buildPythonPackage rec {
+  pname   = "git-pass-helper";
```

Comment:
We put most git related packages into `pkgs/applications/version-management/git-and-tools`. Would this one fit there too?

Reply:
I guess so. Moved.